python-client/giskard/datasets/base/__init__.py

```diff
@@ -615,6 +615,15 @@ def copy(self):
 
         return dataset
 
+    def to_mlflow(self, client=None, run_id=None):
+        import mlflow
+        with tempfile.NamedTemporaryFile(prefix="dataset-", suffix=".csv") as f:
+            self.df.to_csv(f.name)
```

Comment:
why not use `giskard.client.io_utils.save_df` which handles None/empty string differentiation ?